eng/Directory.Build.Data.targets

```diff
@@ -80,7 +80,24 @@
   <PropertyGroup>
     <CentralPackagesFile>$(MSBuildThisFileDirectory)Packages.Data.props</CentralPackagesFile>
     <CentralPackageVersionPackagePath>$(MSBuildThisFileDirectory)Microsoft.Build.CentralPackageVersions\2.0.46\Sdk</CentralPackageVersionPackagePath>
+    <UseProjectReferenceToAzureClients Condition="'$(UseProjectReferenceToAzureClients)' == ''">false</UseProjectReferenceToAzureClients>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(UseProjectReferenceToAzureClients)' == 'true'">
```

Comment:
Because this is in the targets, an override at the SDK level in the build props file would be sufficient to toggle for local use, correct?

Reply:
Yes

Reply:
Definitely you can set this property anywhere in a props or project file or the command line to get this behavior.